### ISSUE TITLE
Modify text explaining the impact of weather on satellite NO2 data

### ISSUE THREAD

**Issue by barrylefer** [NASA-IMPACT/covid-dashboard#325]
-This description for NO2 (copied below) is a bit deceiving. Satellite retrieved NO2 can change due to other factors such as weather or transport, so saying that higher OMI NO2 represents activity levels is not entirely true. It should probably say something like “Darker colors can indicate more NO2 pollution …" without linking to activity. 
 Darker colors indicate higher nitrogen dioxide (NO₂) levels associated and more activity. Lighter colors indicate lower levels of NO₂ and less activity.

**Comment by drewbo** [comment]
@barrylefer sorry picking this one up late. Does more NO2 pollution generally correlate with higher activity (even if there are exceptions)? I'd like to avoid removing the entire link to activity if possible

**Comment by barrylefer** [comment]
Yes. So NO2 is a result of combustion (cars, trucks, power plants, fires, etc).  During the pandemic the electrical use has been almost the same (slightly lower) but the amount of automobile traffic is generally way down as many people are no longer commuting to work. So in most cities the decrease in NO2 is a result of less traffic.
